docs/Projects/LLM-benchmarks/project_description.md

```diff
@@ -0,0 +1,26 @@
+# LLL-Benchmark project
+
+Large Languages Models (LLM's) are becoming increasingly popular in assisting people in
+a variety of tasks. These tasks include, but are not limited to, information retrieval, assisting with
+coding and essay writing. In the context of the government, tasks can include for example supporting FOIA
```

Comment:
I think the abbreviation FOIA is maybe not so well known and therefore weird to introduce it here in one go maybe

Reply:
Agreed, I would have to Google it